### ISSUE TITLE
Can you update the library?

### ISSUE THREAD

**Issue by IriaSomobu** [psyycker/react-translation#6]
Please?

I get this error when adding library via npm:
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: web@0.1.0
npm ERR! Found: react@18.2.0
npm ERR! node_modules/react
npm ERR!   react@"^18.2.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^17.0.0" from @psyycker/react-translation@0.5.0
npm ERR! node_modules/@psyycker/react-translation
npm ERR!   @psyycker/react-translation@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.

```

**Comment by psyycker** [comment]
Hi, I updated the dependencies and peerdeps.
I released everything under version 0.5.1. Hope that'll help.
Closing for now

**Comment by IriaSomobu** [comment]
Big thank you! Works like a charm
